### ISSUE TITLE
Inconsistent syntax highlighting between various types of code blocks

### ISSUE THREAD

**Issue by gendx** [google/comprehensive-rust#343]
Looking at [this page](https://google.github.io/comprehensive-rust/ownership/lifetimes-function-calls.html), it seems that syntax highlighting (and line spacing) is inconsistent between code blocks marked as `editable` vs. `ignore`.

I find the colors somewhat better for `ignore`, but the drawback is that it's not editable, and the extra line spacing is not necessary.

---

`editable` block:

![image](https://user-images.githubusercontent.com/4346404/216648298-059a6e7d-e339-40aa-b156-caa0c0282dfc.png)

---

`ignore` block:

![image](https://user-images.githubusercontent.com/4346404/216648579-685475d4-2200-4b5b-ae22-a20bab268cce.png)

**Comment by mgeisler** [comment]
Yes, you're quite right! I reported this upstream as well: https://github.com/rust-lang/mdBook/issues/1796.

I remember playing around with using the Ace editor for all code (and then toggling the read-only mode for blocks which are not editable). I can probably dig that code out again from somewhere :smile:

**Comment by gendx** [comment]
Glad to see it's already reported upstream :)
